test/index.spec.js

```diff
@@ -83,6 +83,14 @@ describe("babel-preset-env", () => {
         }, Error);
       });
     });
+
+    it("should return the current locally installed Electron version with option 'current'", function() {
+      assert.deepEqual(babelPresetEnv.getTargets({
+        electron: "current"
+      }), {
+        chrome: parseInt(electronToChromiumData.electronToChromium("1.6.1"), 10)
```

Comment:
With requiring just the list of full electron versions, `electronToChromiumVersions["1.6.1"]` would be then be the way to get to this info.